e2-studio-star-rx72n-firmware/libs/rx_session/src/rx_session.c

```
@@ -0,0 +1,308 @@
+
+#if !RX_IS_SIMULATOR
+  (void)tx_mutex_delete(&s_session_mutex);
+#endif
+
+  state->initialized = false;
+
+  rx_log_info(s_tag, "Session deinitialized");
+  return k_rx_ok;
+}
+
+rx_err_t rx_session_next_tx(rx_session_state_t* state, uint16_t* sequence)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_session_next_tx' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)